src/components/BlogForm/NewCard.tsx

```diff
@@ -0,0 +1,177 @@
+      <Stack direction="column" spacing={2} py={2}>
+        <Box
+          sx={{
+            display: "flex column",
+            flex: "space-between",
+            border: "solid black 2px",
+            borderRadius: "14px",
+          }}
+        >
+          <Container>
+            <Controller
+              name="title"
+              control={control}
+              rules={{ required: true }}
+              render={({
+                field: { onChange, value },
+                fieldState: { error },
+              }: any) => (
+                <TextField
+                  fullWidth
+                  label="Title"
+                  multiline
+                  minRows={1}
+                  value={value}
+                  onChange={onChange}
+                  error={!!error}
+                  helperText={!!error && "Title is required"}
+                  size="small"
+                  sx={{ my: "10px" }}
+                />
+              )}
+            />
+
+            <Controller
+              name="desc"
+              control={control}
+              rules={{ required: true }}
+              render={({
+                field: { onChange, value },
+                fieldState: { error },
+              }: any) => (
+                <TextField
+                  fullWidth
+                  label="Description"
+                  multiline
+                  minRows={5}
+                  value={value}
+                  onChange={onChange}
+                  error={!!error}
+                  helperText={!!error && "Body text is required"}
+                  size="small"
+                  sx={{ my: "10px" }}
+                />
+              )}
+            />
+
+            <Controller
+              name="tags"
+              control={control}
+              rules={{ required: true }}
+              render={({
+                field: { onChange, value },
+                fieldState: { error },
+              }: any) => (
+                <TextField
+                  label="Tags"
+                  select
+                  fullWidth
+                  value={value}
+                  onChange={onChange}
+                  error={!!error}
+                  helperText={!!error && "A Tag is required"}
+                  size="small"
+                  sx={{ my: "10px" }}
+                >
+                  {" "}
+                  {tagOptions.map((option) => (
+                    <MenuItem key={option} value={option}>
+                      {option}
+                    </MenuItem>
+                  ))}
+                </TextField>
+              )}
+            />
+          </Container>
+        </Box>
+
+        <input type="file" onChange={handleImageChange} />
+
+        <Button type="submit" variant="contained" sx={{ mb: "20px" }}>
+          Submit
+        </Button>
+      </Stack>
+    </form>
+  );
+};
+
+export default NewCard;
+
+const tagOptions = ["Finance", "Jobs", "Education", "Loose Cannon"];
```

Comment:
move this into the file with other constants (menuOptions etc...) maybe rename that file as blogConstants or something